src/containers/SearchPage/components/results/SearchResult.tsx

```diff
@@ -36,11 +38,11 @@ const SearchResult = ({ result, locale, type, subjects, editingState }: Props) =
   const { t } = useTranslation();
   switch (type) {
     case 'content':
-      return <SearchContent content={result as MultiSearchSummary} locale={locale} />;
+      return <SearchContent content={result as IMultiSearchSummary} locale={locale} />;
```

Comment:
Hadde vært bra om vi fant en måte for å skrive om dette slik at result ikke må castes alle stedene.

Reply:
Doneded :+1: